sdk/freestanding-execution-engine/src/main.rs

```diff
@@ -124,49 +121,45 @@ fn parse_command_line() -> Result<CommandLineOptions, Box<dyn Error>> {
                 .help(
                     "Selects the execution strategy to use: interpretation or JIT (defaults to \
                      interpretation).",
-                )
-                .required(false)
-                .multiple(false)
-                .default_value("jit"),
+                ),
         )
         .arg(
             Arg::with_name("dump-stdout")
                 .short("o")
                 .long("dump-stdout")
                 .help("Whether the contents of stdout should be dumped before exiting")
-                .required(true)
-                .value_name("BOOLEAN")
-                .default_value(&default_dump_stdout),
+                .value_name("BOOLEAN"),
         )
         .arg(
             Arg::with_name("dump-stderr")
                 .short("e")
                 .long("dump-stderr")
                 .help("Whether the contents of stderr should be dumped before exiting")
-                .required(true)
-                .value_name("BOOLEAN")
-                .default_value(&default_dump_stderr),
+                .value_name("BOOLEAN"),
         )
         .get_matches();
 
     info!("Parsed command line.");
 
-    let execution_strategy = if let Some(strategy) = matches.value_of("execution-strategy") {
-        if strategy == "interp" {
+    let execution_strategy = {
+        let strategy = matches.value_of("execution-strategy").unwrap_or("jit");
+        match strategy {
+            "interp" => {
             info!("Selecting interpretation as the execution strategy.");
```

Comment:
Indentation is off starting from here

Reply:
I've fixed that, I hope. But I did `git push --force`, which perhaps wasn't the best way. Or perhaps it doesn't matter.